src/client/Microsoft.Identity.Client/Platforms/Android/Broker/BrokerRequest.cs

```diff
@@ -121,6 +121,12 @@ private static BrokerRequest FromAuthenticationParameters(AuthenticationRequestP
                 br.Claims = _authenticationRequestParameters.Claims;
             }
 
+            if (_authenticationRequestParameters.RequestContext.ServiceBundle.Config.ClientCapabilities?.Any() == true)
+            {
+                br.Claims = ClaimsHelper.GetMergedClaimsAndClientCapabilities(br.Claims,
```

Comment:
no need to merge, `ClaimsAndClientCapabilities` is already merged. so instead of block 119-122 and 124-129, you can have a single 

`if (ClaimsAndClientCapabilities.Any())
{
 br.Claims = ClaimsAndClientCapabilities;
}

Reply:
Fixed.